app/assets/javascripts/announcements.js

```diff
@@ -1,29 +1,36 @@
-function hideAnnouncement(announcement_key) {
-  createCookie(announcement_key, "hidden", 365);
-  $("#announcement").slideUp();
-}
+$(function() {
 
-// Utility scripts from http://www.quirksmode.org/js/cookies.html
-function createCookie(name, value, days) {
-  if (days) {
-    var date = new Date();
-    date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
-    var expires = "; expires=" + date.toGMTString();
-  } else var expires = "";
-  document.cookie = name + "=" + value + expires + "; path=/";
-}
+  var announcement = $('#announcement');
+  var hideAnnouncement = $('#announcement .hide > a');
```

Comment:
What do you think about using `data-attributes` for this? That way we are not relying on the HTML. For example we could add `data-behaviour="hide-announcement"` to the link.